src/vorta/utils.py

```diff
@@ -169,20 +169,21 @@ def sort_sizes(size_list):
     return final_list
 
 
-def pretty_bytes(size):
-    """from https://stackoverflow.com/questions/12523586/
-            python-format-size-application-converting-b-to-kb-mb-gb-tb/37423778"""
+def pretty_bytes(size, metric=True, sign=False, precision=1):
```

Comment:
Nice improvements to `pretty_bytes()`. But you don't use the new `metric` and `sign` args anywhere?

Reply:
I also removed the SO reference, since the function substantially changed.

Reply:
In an earlier of my changes, I was using some of the enhancements to pretty_bytes, but after adding the test code and realizing the displayed tree did not match exactly with the non-json-lines version, I didn't use it, and forgot to revert those changes.  There  are two 'bug-fixes', but I don't think either one would be triggered. 1. better handling of something larger than 1000TB - my version with the loop index check will at least attempt to print the number using 'TB' units, instead of NaN (due to array index exception). 2. better handling of negative 'size' - print correct value with appropriate units instead of using base/unscaled units ('B'). Currently, I think the tree only shows the '+' change in file content, and ignores the '-' change in file content, so #2 probably isn't triggered either. And then a 3rd 'issue', is when a number is close to the threshold of using the next larger units, but doesn't use them, but then due to rounding prints as '1000' ...  Example: size = 999.95 KB -- it is not large enough to go through the loop twice and use 'MB' units, but when printed, it is rounded up and prints as '1000.0 KB' instead of  '1.0 MB'. It seems like the preferred display would limit the significand to 4 total digits and not sometimes round up and display 5 digits.

Reply:
I could split off the pretty_bytes changes as a separate PR and indicate the issues it addresses.

Reply:
Nah, it's fine. The new args might be useful in the future and it fixes some issues as you say.